docs/operations/replication.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "How To Use Volume Replication"
+linkTitle: How To Use Volume Replication
+---
+
+## Overview
+
+> 💡 For more information on Ondat's Replication feature, review the  [Replication](/docs/concepts/replication)  feature page.
+
+### Example - Enable Volume Replication Through a `PersistentVolumeClaim` Definition
+
+The following guidance below will demonstrates how to use Ondat's Volume Replication  through a  `PersistentVolumeClaim`  (PVC) definition.
+-   The instructions will enable volume replication on a PVC with the label »  `storageos.com/replicas=1` - which will create `1` master volume and `1` replica volume respectively.
```

Comment:
Is the **»** intended to be there?

Reply:
It looks like it should be `>>` rather than `»`

Reply:
yes - I was use it point to specific items, but I am happy to remove it